src/modules/fw_att_control/fw_att_control_main.cpp

```diff
@@ -938,12 +955,12 @@ FixedwingAttitudeControl::task_main()
 				float airspeed;
 
 				/* if airspeed is non-finite or not valid or if we are asked not to control it, we assume the normal average speed */
-				const bool airspeed_valid = PX4_ISFINITE(_sub_airspeed.get().indicated_airspeed_m_s)
-							    && ((_sub_airspeed.get().timestamp - hrt_absolute_time()) < 1e6);
+				const bool airspeed_valid = PX4_ISFINITE(_airspeed.indicated_airspeed_m_s)
+							    && (fabs((double)_airspeed.timestamp - (double)hrt_absolute_time()) < 1.0e6);
```

Comment:
Why use doubles here? I would change this to `hrt_elapsed_time(&_airspeed.timestamp)`.

Reply:
yes - good call. please see the comment in #8899